### PR TITLE
Local dataset Noorderplantsoenbuurt, changes correct previous errors and represent more detailed analysis

### PR DESCRIPTION
**Pull request by DorinevanderVlies**
Most of the analysis is based on final demand and cadastre data.
The supply side is NL-2013-scaled.

To be improved in the future:
- The demand of the buildings section is estimated based on final demand data of 5 out of 60 buildings. More data is requested.
- Buildings use for appliances not yet corrected for savings by motion and daylight control. Will be corrected once more detailed data regarding final demand of buildings is processed.
- Potential PV on houses and buildings is currently scaled to Paddepoel Noord (PPN) data. 
- Exact number of inhabitants (currently using 1.7 * households, based on PPN).
- Number of old and new houses and buildings might have to be adjusted once more detailed data is received.

### CODE REVIEW

**Comment by ChaelKruip** [comment]
This test fails:

```
Failures:

  1) Validating datasets: npb
     Failure/Error: expect(document).to be_valid
       expected that :npb would be valid
       Diff:
       @@ -1 +1,2 @@
       +:init => ["contains inputs belonging to the cooling_buildings share group, but the values sum to 1.0, not 100"]
     # ./spec/validate_documents_spec.rb:32:in `block (4 levels) in <top (required)>'
```

@DorinevanderVlies can you check that this share group is indeed correctly set?

File: datasets/npb/npb.derived.ad [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+- init.buildings_chp_engine_biogas_share = 0
+- init.buildings_collective_chp_network_gas_share = 0
+- init.buildings_collective_chp_wood_pellets_share = 0
+- init.buildings_collective_geothermal_share = 0
+- init.buildings_heat_network_connection_steam_hot_water_share = 100
+# Cooling
+- init.buildings_useful_demand_cooling = 0
+- init.buildings_cooling_airconditioning_share = 0
+- init.buildings_cooling_collective_heatpump_water_water_ts_electricity_share = 0
+- init.buildings_cooling_heatpump_air_water_network_gas_share = 1
```
Comment (ChaelKruip) [inline datasets/npb/npb.derived.ad]: I think this should be 100

Comment (DorinevanderVlies) [inline datasets/npb/npb.derived.ad]: Indeed

**Comment by ChaelKruip** [comment]
Nice work @DorinevanderVlies!

**Pull request merged**